packages/lodestar/src/network/reqResp.ts

```diff
@@ -240,7 +244,8 @@ export class ReqResp extends (EventEmitter as IReqEventEmitterClass) implements
     peerId: PeerId,
     method: Method,
     encoding: ReqRespEncoding,
-    request?: RequestBody
+    request?: RequestBody,
+    signal?: AbortSignal
```

Comment:
can we access `signal` directly from `this.controller`? this helps avoid adding a new param here.

Reply:
done